pkg/scheduler/framework/preemption/preemption.go

```diff
@@ -238,15 +242,22 @@ func (ev *Evaluator) findCandidates(ctx context.Context, allNodes []*framework.N
 		return nil, nil, err
 	}
 
-	offset, numCandidates := ev.GetOffsetAndNumCandidates(int32(len(potentialNodes)))
+	offset, candidatesNum := ev.GetOffsetAndNumCandidates(int32(len(potentialNodes)))
 	if loggerV := logger.V(5); logger.Enabled() {
 		var sample []string
-		for i := offset; i < offset+10 && i < int32(len(potentialNodes)); i++ {
-			sample = append(sample, potentialNodes[i].Node().Name)
+		if sampleLength >= len(potentialNodes) {
+			for _, node := range potentialNodes {
+				sample = append(sample, node.Node().Name)
```

Comment:
Wouldn't this include the nodes before the offset?

Reply:
Yes, that's intended. However, I changed the logic here as suggested.